### PR TITLE
feat: new email template for listing opportunity

### PR DESCRIPTION
**Pull request by emilyjablonski**
# Pull Request Template

## Issue Overview

This PR addresses #348 

- [x] This change addresses the issue in full

## Description

Adds the new email template. The icon doesn't match 1-1 because none of our current Doorway emails have that styling. The email just has mock data for now, adding translations and listing data would be part of implementation.

## How Can This Be Tested/Reviewed?

I updated [this line](https://github.com/metrotranscom/doorway/blob/3866942e20e218be0b6a777cf6f64051d4b15452/backend/core/src/auth/services/user.service.ts#L410) to `await this.emailService.listingOpportunity(user, dto.appUrl)` and went through the forgot password flow in partners.

## Checklist:

- [x] My code follows the style guidelines of this project
- [ ] I have added QA notes to the issue with applicable URLs
- [x] I have performed a self-review of my own code
- [x] I have reviewed the changes in a desktop view
- [x] I have reviewed the changes in a mobile view
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] I have assigned reviewers
- [ ] I have run `yarn generate:client` and/or created a migration if I made backend changes that require them
- [x] My commit message(s) is/are polished, and any breaking changes are indicated in the message and are well-described
- [ ] Commits made across packages purposefully have the same commit message/version change, else are separated into different commits

## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.

### CODE REVIEW

**Review by ColinBuyck — APPROVED** [review]
Looking good, just one small question, the bold on the row name seems to be 700 in the Figma rather than 600 but unsure if 600 was chosen to follow existing patterns or something else. Either way works for me!

File: backend/core/src/email/email.service.ts [hunk 1]
```diff
@@ -214,6 +214,39 @@ export class EmailService {
     )
   }
 
+  public async listingOpportunity(user: User, appUrl: string) {
```
[the remark this answers is not in the document]
Comment (emilyjablonski) [reply backend/core/src/email/email.service.ts:217]: As far as I know this wouldn't be temporary and would be what we call similar to the other emails unless I'm misunderstanding how we're going to work w GovDelivery

[the remark this answers is not in the document]
Comment (ludtkemorgan) [reply backend/core/src/email/email.service.ts:217]: Sorry, temporary was not the right word. More that we need to replace the call with mock data to the real logic. So a todo would suffice

[the remark this answers is not in the document]
Comment (emilyjablonski) [reply backend/core/src/email/email.service.ts:217]: oh yes absolutely!

**Review by ludtkemorgan — APPROVED** [review]

**Pull request merged**